### ISSUE TITLE
Cannot Use Remote Debugger

### ISSUE THREAD

**Issue by jhwheeler** [facebook/react-native#22368]
- [ Yes ] Review the documentation: https://facebook.github.io/react-native
  - [ Yes ] Search for existing issues: https://github.com/facebook/react-native/issues
  - [ Yes ] Use the latest React Native release: https://github.com/facebook/react-native/releases

## Environment
React Native Environment Info:
    System:
      OS: macOS 10.14.1
      CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
      Memory: 91.83 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.6.0 - ~/.nvm/versions/node/v10.6.0/bin/node
      Yarn: 1.12.3 - ~/.nvm/versions/node/v10.6.0/bin/yarn
      npm: 6.4.1 - ~/.nvm/versions/node/v10.6.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
    IDEs:
      Android Studio: 3.1 AI-173.4819257
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.5.0 => 16.5.0
      react-native: https://github.com/expo/react-native/archive/sdk-31.0.1.tar.gz => 0.57.1

The app is using Expo v31.0.3 and expo-cli 2.4.0.

## Description
The remote debugger does not work. The app works fine without the remote debugger active in both Android Emulator and iOS Simulator. Production mode is not active. 

<img width="1680" alt="screenshot 2018-11-21 at 15 30 43" src="https://user-images.githubusercontent.com/23257850/48844800-94ff1f80-eda3-11e8-8b2e-1650cb481ae6.png">

Here is the error message:

<img width="1305" alt="screenshot 2018-11-21 at 15 31 09" src="https://user-images.githubusercontent.com/23257850/48844936-e5767d00-eda3-11e8-8844-69e01f9d225b.png">

Changing the url to use localhost instead of the local IP does not fix it (as suggested in similar issue here: https://github.com/facebook/react-native/issues/17618. Furthermore we are not using xip.io).

## Reproducible Demo
Not sure how to reproduce, as I can get the remote debugger working with a clean `react-native init` project. The repo is private, so I cannot share a link to it.

**Comment by radko93** [comment]
I had a similar issue and I restarted simulator/packager/computer/browser and it was gone. DId you try it?

**Comment by jhwheeler** [comment]
@radko93 Yes, I have. I've also tried removing and reinstalling node_modules, as well as removing the global expo-cli package. 

This issue is being experienced by all 3 of the developers working on this app, and only appeared about a week or so ago.

**Comment by matthargett** [comment]
With newer Chrome and react-devtools, you *have* to use localhost (or the IP addresses involved have to be the same) in order to bypass CORS checking. This has come up quite a few times with teams I work with.

**Comment by kelset** [comment]
@matthargett do you want the issue to be reopened? Or there is some PR we can point people to?

**Comment by matthargett** [comment]
no, just wanted to leave a workaround note for folks who end up here from a web search :)
